profiling/src/profile/mod.rs

```diff
@@ -1034,4 +1098,106 @@ mod api_test {
 
         assert_eq!(endpoints_stats, expected_endpoints_stats);
     }
+
+    #[test]
+    fn local_root_span_id_label_as_i64() {
```

Comment:
Minor: Should we perhaps test using u64 and with one of the ids being UINT_MAX, just to doublecheck that it's handled correctly?